### PR TITLE
Electric power

### PR DESCRIPTION
**Pull request by erdum**
### Describe your change: ###
Used code from ohms_law algorithm, but now for power calculation 

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by erdum** [comment]
@cclauss hello, sir please debug it and give me some tips and instructions, some lines in code are longer then 79 so what should i need to do????

**Comment by cclauss** [comment]
Maximum line length is 88 characters.

File: electronics/electric_power.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# https://en.m.wikipedia.org/wiki/Electric_power
+
+
+def electric_power(voltage: float, current: float, power: float) -> float:
+    """
+    This function can calculate any one of the three (voltage, current, power), fundamental value of electrical system.
```
Comment (cclauss) [inline electronics/electric_power.py:6]: This line needs to be wrapped at 88 characters max.

File: electronics/electric_power.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if (voltage, current, power).count(0) != 1:
+        raise ValueError("Only one argument must be 0")
+    elif power < 0:
+        raise ValueError("Power cannot be negative in any electrical/electronics system")
+    elif voltage == 0:
+        return {'voltage': power / current}
+    elif current == 0:
+        return {'current': power / voltage}
+    elif power == 0:
+        # I created this variable because i want to remove negative sign from the power value
```
Comment (cclauss) [inline electronics/electric_power.py:24]: This line needs to be wrapped at 88 characters max.

File: electronics/electric_power.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        result = (float(voltage * current))
+        if result < 0:
+            result = result * -1
+        return {'power': result}
```
Comment (cclauss) [inline electronics/electric_power.py:28]: Please use the builtin function abs()

Comment (erdum) [reply electronics/electric_power.py:28]: Ok

**Comment by erdum** [comment]
@cclauss all done sir.

File: electronics/electric_power.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    """
+    This function can calculate any one of the three (voltage, current, power),
+    fundamental value of electrical system.
+    examples are below:
+    >>> electric_power(voltage=0, current=2, power=5)
+    {'voltage': 2.5}
+    >>> electric_power(voltage=2, current=2, power=0)
+    {'power': 4.0}
+    >>> electric_power(voltage=-2, current=3, power=0)
+    {'power': 6.0}
```
Comment (cclauss) [inline electronics/electric_power.py:14]: Need to test:
1. No zeros
2. Two zeros
3. Negative numbers for power
4. Floating-point numbers
```suggestion
    >>> electric_power(voltage=-2, current=3, power=0).name
    'power'
    >>> electric_power(voltage=-2, current=3, power=0).value
    6.0
```

File: electronics/electric_power.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# https://en.m.wikipedia.org/wiki/Electric_power
+
```
Comment (cclauss) [inline electronics/electric_power.py:2]:
```suggestion

from collections import namedtuple

result = namedtuple("result", "name value")

```

File: electronics/electric_power.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+def electric_power(voltage: float, current: float, power: float) -> float:
+    """
+    This function can calculate any one of the three (voltage, current, power),
+    fundamental value of electrical system.
+    examples are below:
+    >>> electric_power(voltage=0, current=2, power=5)
+    {'voltage': 2.5}
```
Comment (cclauss) [inline electronics/electric_power.py:10]:
```suggestion
    result(name='power', value=2.5)
```

File: electronics/electric_power.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    {'power': 6.0}
+    """
+    if (voltage, current, power).count(0) != 1:
+        raise ValueError("Only one argument must be 0")
+    elif power < 0:
+        raise ValueError(
+            "Power cannot be negative in any electrical/electronics system"
+        )
+    elif voltage == 0:
+        return {"voltage": power / current}
```
Comment (cclauss) [inline electronics/electric_power.py:23]:
```suggestion
        return result("voltage", power / current)
```

**Review by cclauss — CHANGES_REQUESTED** [review]
Let's spice up the tests and let's return a namedtuple instead of a dict.  Namedtuples are more optimized than dicts and they provides helpful names.

File: electronics/electric_power.py (hunk 4 above)
Comment (erdum) [reply electronics/electric_power.py:14]: Okay, should i also do this on ohms_law as well??

Comment (cclauss) [reply electronics/electric_power.py:14]: Let's get this PR landed ___before___ you modify Ohm's law...

Coincidentally, @rhettinger (who wrote the namedtuple) just tweeted about Ohm's law...
https://twitter.com/raymondh/status/1331850115674345473

**Comment by erdum** [comment]
@cclauss sir run this code and you will see that with floating point values i will give you very long decimal value output so my question is how to wrapped it, only for 3 values after decimal

**Comment by erdum** [comment]
@cclauss I figured out, now all done???

**Review by cclauss — APPROVED** [review]
Nice!

**Pull request merged**